Examples/Swift/ViewController.swift

```diff
@@ -369,6 +369,14 @@ extension ViewController: NavigationViewControllerDelegate {
         navigationViewController.present(confirmationController, animated: true, completion: nil)
         return false
     }
+    
+    
+    // Called when the user hits the exit button.
+    // If implemented, you are responsible for also dismissing the UI.
+    func navigationViewControllerDidCancelNavigation(_ navigationViewController: NavigationViewController) {
+        print("The user has exited")
```

Comment:
Is this necessary?

Reply:
Shrug, was just making it obvious to the developer. can remove.

Reply:
Dealer's choice.